apps/src/code-studio/components/playzone.story.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PlayZone from './playzone';
-import CreateSomething from './lessonExtras/CreateSomething';
 
-export default storybook => {
-  storybook.storiesOf('PlayZone', module).addStoryTable([
-    {
-      name: 'Default',
-      story: () => <PlayZone lessonName="Test Lesson" onContinue={() => {}} />
-    },
-    {
-      name: 'Create something',
-      story: () => <CreateSomething />
-    }
-  ]);
+export default {
+  title: 'PlayZone',
+  component: PlayZone
+};
+
+// Template
+const Template = args => <PlayZone {...args} />;
+
+// Stories
+export const Default = Template.bind({});
+Default.args = {
+  lessonName: 'Test Lesson',
+  onContinue: () => {}
```

Comment:
i know this is from the original story, but we usually use the `action` addon so that it's clear in storybook that clicking 'continue' does something, it's just mocked out in storybook (i hope that makes sense?)

so we could import the action addon and replace our no-op function with `action('continuing...')` (the string can be whatever message we want -- it just gets logged to the 'Action' console in storybook so the user sees it

Reply:
(same thought with `assetChosen` in IconLibrary)